crates/cxx-qt/src/qobject.rs

```diff
@@ -0,0 +1,22 @@
+// SPDX-FileCopyrightText: 2025 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Ben Ford <ben.ford@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+//! QObject module
+
+#[cxx::bridge]
+mod ffi {
+    unsafe extern "C++" {
+        // TODO! Implement QObject wrapper properly
```

Comment:
Right and then we reach the `QObject::thread() -> QThread` method, which @LeonMatthesKDAB might help with that threading idea :-)

Reply:
I should note that we likely will implement QObject via an extension trait in cxx-qt-lib, as many of the methods rely on types that we only get via cxx-qt-lib, and we don't want cxx-qt to depend on -lib.

Reply:
Nice :-)